migrator/src/main/java/uk/nhs/digital/ps/migrator/task/ImportableItemsFactory.java

```diff
@@ -38,7 +38,8 @@ public static DataSet toDataSet(final PublishingPackage exportedPublishingPackag
         return new DataSet(
             parentFolder,
             exportedPublishingPackage.getTitle(),
-            exportedPublishingPackage.getTitle()
+            exportedPublishingPackage.getTitle(),
```

Comment:
Duplicate getTitle?

Reply:
At the moment the dataset name and title are both set to the same thing as we only have this value from the exported CI data.